angular.json

```diff
@@ -96,12 +97,16 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
```

Comment:
why having the same code  like code line 31 try removing this one and see its working or not once